README.md

```diff
@@ -168,6 +168,19 @@ ros2 launch zivid_samples zivid_camera_with_serial_number.launch serial_number:=
 > Read more about [firmware update in our knowledgebase][firmware-update-kb-url].
 > This parameter is optional, and by default it is true.
 
+`color_space` (string, default: "srgb")
+> Specify the color space to use when publishing point clouds and images. Valid values:
+>
+>  - `srgb`: Publish in the sRGB color space. The sRGB color space is suitable for showing an image
+>    on a display for uman viewing. It is easier to see details in darker areas of an image in sRGB
+>    than in linear RGB, as more of the dynamic range is dedicated to darker colors. This format is
+>    assumed by default by most monitors and should be used when displaying an image.
+>  - `linear_rgb`: Publish in linear RGB color space. Linear RGB is suitable as input to computer
+>    vision algorithms.
+>
+> In particular, this parameter affects the data published over the [color/image_color](#colorimage_color) and
```

Comment:
Can link to https://support.zivid.com/en/latest/reference-articles/color-spaces-and-output-formats.html here?